.github/workflows/build_flang_windows_x64.yml

```diff
@@ -0,0 +1,85 @@
+      - '**/.github/workflows/build_flang_arm64.yml'
+
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
+      - '**/.github/workflows/build_flang_arm64.yml'
+
+jobs:
+  build_flang:
+
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        llvm_branch: [ release_15x ]
+
+    steps:
+      - name: Clean Workspace
+        run: Remove-Item * -Recurse -Force
+
+      # Checks-out your repository under $GITHUB_WORKSPACE, so the job can access it
+      - uses: actions/checkout@v3
+
+      - name: Setup Windows
+        uses: llvm/actions/setup-windows@main
+        with:
+          arch: amd64
+ 
+      - name: Setup Ninja
+        uses: llvm/actions/install-ninja@main
+
+      - name: Setup Graphviz for Doxygen
+        run: choco install graphviz
+
+      - name: Setup Sphinx
+        run: pip install sphinx
+
+      - name: Display path
+        run: write-output ${env:PATH}
+
+      - name: Check tools
+        run: |
+          git --version
+          cmake --version
+          clang --version
+          ninja --version
+
+      # Download artifacts for the classic-flang-llvm-project-related builds (all toolchains)
+      - name: Download and extract LLVM artifact
+        run: |
+          cd ../..
+          Invoke-WebRequest -Uri https://api.github.com/repos/kaadam/classic-flang-llvm-project/actions/workflows/pre-compile_llvm.yml/runs -OutFile runs_llvm.json
+          Invoke-WebRequest "$(jq -r '.workflow_runs[0].artifacts_url?' runs_llvm.json)" -OutFile artifacts_llvm
+
+          $i=0
+          # Keep checking older builds to find the right branch and correct number of artifacts
+          while ( ( "$(jq -r '.total_count?' artifacts_llvm)" -ne "1" ) -or `
+                  ( "$(jq -r --argjson i $i ".workflow_runs[$i].head_branch" runs_llvm.json)" -ne "${{ matrix.llvm_branch }}") -and `
+                  ($i -lt 2)
+                )
+          {
+            write-output "No artifacts or wrong branch in build $i, counting from latest"
+            $i += 1
+            Invoke-WebRequest -Uri "$(jq -r --argjson i $i ".workflow_runs[$i].artifacts_url" runs_llvm.json)" -OutFile artifacts_llvm
+          }
+          $url="$(jq -r '.artifacts[] | select(.name==\"llvm_build_win64_clangcl_${{ matrix.llvm_branch }}\") | .archive_download_url' artifacts_llvm)"
+          write-output "$($url)"
+          $artifactPath = "$pwd\llvm_build_win64_clangcl_${{ matrix.llvm_branch }}.zip"
+          Invoke-RestMethod -Method Get -Uri $url -OutFile $artifactPath -Headers @{ "Authorization" = "Bearer ${{ secrets.GITHUB_TOKEN }}" }
+          unzip -o llvm_build_win64_clangcl_${{ matrix.llvm_branch }}.zip
+          & 7z x "$pwd\llvm-build-win64.7z" -o"$pwd\classic-flang-llvm-project\" -y
+          write-output "$(Get-ChildItem)"
+
+      - name: Build Flang & Libpgmath
+        run: |
+          python .\scripts\build_flang.py -d build -p "$(pwd)\..\..\classic-flang-llvm-project\classic-flang-llvm-project\build\"
```

Comment:
Why doesn't this step install flang? The step name should be `Build and install flang & libpgmath` to be consistent with other jobs.

Reply:
Basically it does. The build script will install pgmath and flang into '-p "$(pwd)\..\..\classic-flang-llvm-project\classic-flang-llvm-project\build\'.  But yes, a clean directory would be better.

Reply:
I see, the script performs installation unconditionally.

Reply:
Done